src/lib/forms-schema/index.ts

```diff
@@ -479,6 +480,10 @@ export const WorkflowEventSchema = Joi.object().keys({
         excludeAttachments: Joi.boolean().default(false),
         ...pdfSubmissionEventConfiguration,
       }),
+    })
+    .when('type', {
+      is: 'CIVIC_REC_COMPLETE_CHECKOUT',
+      then: Joi.any().strip(),
```

Comment:
Better to make this an empty object so that we don't have to migrate existing events if we add some configuration later.

Reply:
It should be returning an empty object, as discussed, I will add some unit tests and refine this if needed :)